genesis/streaming.py

```diff
@@ -132,7 +131,8 @@ def __init__(
             #   - generate a random groupid
             #   - disable committing (so we never commit the random groupid)
             #   - start reading from the earliest (smallest) offset
-            import getpass, random
+            import getpass
+            import random
```

Comment:
It would be good to push these to the top of the file -- I was just being lazy here.

Generally, imports should not be function or class level _unless_ there's a good reason to defer their loading (e.g., they're packages which may not be installed, or packages whose import takes awhile and shouldn't be done unless the user requested something that needs them).